lib/matplotlib/axis.py

```diff
@@ -635,6 +635,9 @@ class Axis(martist.Artist):
         The minor ticks.
     """
     OFFSETTEXTPAD = 3
+    # The class used to create tick instances.
+    # Must be overwritten in subclasses.
+    _TICK_CLASS = None
```

Comment:
I don't think this needs to be uppercased.

Reply:
Lowercased.

IMHO It's a bit simpler if we define the attribute in the base class. A not-implemented property as a placeholder that is overwritten has a slightly nicer error handling but is more difficult to understand. Since there is only one usage where we need the error handling, let's keep it simple.